src/u-blox_config_keys.h

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
I think this should have a header block - I hit compilation issues without it when referencing from multiple other files:
```
#ifndef __u_blox_config_keys_h__
#define __u_blox_config_keys_h__
```

Reply:
Indeed! I'll get it added.

Reply:
I agree! ;-)